state-sync/state-sync-driver/src/storage_synchronizer.rs

```diff
@@ -207,6 +211,16 @@ impl<
             chunk_executor.clone(),
             error_notification_sender.clone(),
             executor_listener,
+            ledger_updater_notifier,
```

Comment:
one orthogonal question. would this also cover the genesis ledger data since  I assume there is no execution done for version 0?

Reply:
Correct -- there is no execution for the genesis data, so it doesn't even enter this flow 😄 'cc @msmouse